docs/working-with-cdk-for-terraform/terraform-functions.md

```diff
@@ -0,0 +1,27 @@
+# Terraform Functions
+
+Terraform provides a set of functions that can be used through the CDK for Terraform.
+
+```ts
+import { Fn } from "cdktf";
+
+new vpc() = new VPC(this, "vpc", {});
+new LoadBalancer(this, "lb", {
+  name: "main-lb",
+  subnet: Fn.Network.cidrsubnet(
+    Fn.Collection.element(vpc.listOfSubnets, 0),
+    4,
+    2
+  ),
+});
+```
+
+To explore the functionality either check out the [Terraform Documentation](https://www.terraform.io/docs/language/functions/index.html) or use your editor autocompletion on the `Fn` object.
```

Comment:
No, I mean using the Token system to get a token of a different type. I.E. using the `coalesce` function is a list of numbers will return `IResolvable` rather than `number`.